scripts/ci/manage-devsandbox-dashboard.sh

```
@@ -0,0 +1,197 @@
+    if ! oc get project ${DEVSANDBOX_DASHBOARD_NS} >/dev/null 2>&1; then
+        echo "Creating namespace ${DEVSANDBOX_DASHBOARD_NS}"
+        oc new-project ${DEVSANDBOX_DASHBOARD_NS} >/dev/null 2>&1 || true
+    else
+        echo "Namespace ${DEVSANDBOX_DASHBOARD_NS} already exists"
+    fi
+    oc project ${DEVSANDBOX_DASHBOARD_NS} >/dev/null 2>&1
+}
+
+set -e
+
+read_arguments $@
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)